vividus-extension-selenium/src/main/java/org/vividus/ui/session/AbstractSessionLinkPublisher.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package org.vividus.ui.report;
+package org.vividus.ui.session;
```

Comment:
it's still not about sessions

Reply:
here we have link publisher to session, setting of status for session, if we open browser stack we see list of sessions to which we publish links and statuses, the same at some degree is applicable for lambdatest